package.json

```diff
@@ -24,7 +24,7 @@
     "deep-equal": "^1.0.1",
     "dlnacasts": "^0.1.0",
     "drag-drop": "^2.12.1",
-    "electron": "1.4.0",
+    "electron": "^1.3.5",
```

Comment:
I think we used to use this field in a script in a way where the `^` would break things, but I just checked and we don't do that anymore.

But I just realized a bigger issue: `^1.3.5` will install anything between 1.3.5 and 2.0.0 (exclusive). So, this still installs 1.4.0 👊